internal/pkg/controllers/profile/profile.go

```diff
@@ -149,8 +150,13 @@ func getProfilePath(profileName string, cfg *corev1.ConfigMap) (string, error) {
 		return "", errors.New(errConfigMapWithoutName)
 	}
 
+	if cfg.ObjectMeta.Namespace == "" {
+		return "", errors.New(errConfigMapWithoutNamespace)
```

Comment:
I don’t think that this is possible in general.

Reply:
Actually, yeah. I think it defaults to `default`.

I guess that makes the check for `name` being enpty moot as well. Cause I think name can't be empty either.

Reply:
Yep, empty namespaces should be forbidden by the API server.